src/plugins/vis_type_drilldown/opensearch_dashboards.json

```
@@ -13,5 +13,6 @@
     "data",
     "visDefaultEditor"
   ],
-  "optionalPlugins": []
+  "optionalPlugins": [],
+  "requiredBundles": ["opensearchDashboardsUtils", "opensearchDashboardsReact", "home"]
```

No need to add these required bundles, as `opensearchDashboardsReact` is already in the `requiredPlugins` section. We may eventually decide to move it to `requiredBundles`, but it should only be in one or the other.
```suggestion
  "optionalPlugins": [],
```